model/http-common.yaml

```diff
@@ -14,7 +14,8 @@ groups:
         examples: ['timeout', 'name_resolution_error', '500']
```

Comment:
Please add an example of an exception class name like it should be represented in this attribute.